### ISSUE TITLE
Range of Smoothness

### ISSUE THREAD

**Issue by DelinQu** [SuTanTank/BundledCameraPathVideoStabilization#8]
Dear SuTanTank:
I tried runing `run.m` to stable frames but the performance is not so good. So I adjusted the hyperparameter `Smoothness` and it worked when setting   `Smoothness = 10`.
By reading your code I found a line:
```matlab
Smoothness = 1;                 % Adjust how stable the output is, 0.5 - 3 is OK
```
Its range is beyond the size you set, I am pretty confused with that. How did you decide the range of this parameter is `0.5 - 3` ?
can I set it much more higher?

Thank you again :smiley_cat: 
---
- Smoothness = 1:
![Undistorted_Asap1](https://user-images.githubusercontent.com/60593268/146919522-0478744f-9672-47ed-9acd-88f91d6906aa.png)

- Smoothness = 10:
![Undistorted_Asap10](https://user-images.githubusercontent.com/60593268/146919660-7e535862-182a-4c26-bb6b-601ae0d4b3fa.png)

**Comment by SuTanTank** [comment]
Yes, all parametres are adjustable, and the range is just for reference. If
you understand the math behind, you can find the best combination for your
use case.

This work is too old and I have forgot the details.

regards,

Delin Qu ***@***.***> 于 2021年12月21日周二 19:04写道：
